test/influxdb_v2/deleter/predicate_test.exs

```diff
@@ -0,0 +1,122 @@
+defmodule Instream.InfluxDBv2.Deleter.PredicateTest do
+  use ExUnit.Case, async: true
+
+  @moduletag :"influxdb_include_2.x"
+
+  alias Instream.TestHelpers.TestConnection
+
+  defmodule EmptyTagSeries do
+    use Instream.Series
+
+    series do
+      measurement "empty_predicate_tags"
+
+      tag :filled
+      tag :defaulting, default: "default_value"
+      tag :empty
+
+      field :value
+    end
+  end
+
+  describe "testing" do
+    setup do
+      data_origin = ~U[2021-01-01T00:00:00Z]
```

Comment:
Maybe inline this variable into the delete calls like done in `Instream.Log.DefaultLoggerTest`?

As the value is in no way dynamic and does not matter as long as it is "some point of time in the past" a `setup` function for it may suggest there is something more happening.

Reply:
inlined, thanks